optimum/intel/openvino/configuration.py

```diff
@@ -669,23 +669,20 @@ def post_init(self):
         if self.bits != 8:
             raise ValueError(f"Only support 8-bit for static quantization but found {self.bits}")
 
-        if self.dataset is not None:
-            if self.dataset not in PREDEFINED_SPEECH_TO_TEXT_DATASETS:
-                raise ValueError(
-                    f"You have entered the following string value for dataset: {self.dataset}. But it is not supported."
-                    f" Currently you can only choose {list(PREDEFINED_SPEECH_TO_TEXT_DATASETS.keys())}."
-                )
-
         if self.smooth_quant_alpha is not None and not (0 <= self.smooth_quant_alpha <= 1):
             raise ValueError(
                 f"SmoothQuant alpha parameter must be in range [0, 1], but found {self.smooth_quant_alpha}"
             )
 
-        if self.weight_format != "int8":
-            raise ValueError("Only 'int8' weight format is currently supported.")
-
-        if self.activation_format != "int8":
-            raise ValueError("Only 'int8' activation format is currently supported.")
+        if not self.sym:
+            if self.activation_format != "int8":
+                raise ValueError(
+                    f"Asymmetric quantization can not be performed in {self.activation_format} activation format."
+                )
+            if self.weight_format != "int8":
+                raise ValueError(
+                    f"Asymmetric quantization can not be performed in {self.weight_format} weight format."
+                )
```

Comment:
I would suggest to initialize `sym` as `True` inside `OVQuantizatioConfig` constructor if `fp8` mode is selected. This option is intended to be used with `int` data types and does not quite make sense with `fp8` data types. Also, this way `--sym` won't be needed to be specified every time `fp8` modes are used. 

cc @AlexKoff88